promql/promqltest/test.go

```diff
@@ -404,6 +449,8 @@ func (t *test) parseEval(lines []string, i int) (int, *evalCmd, error) {
 		cmd.info = true
 	}
 
+	var allowExpectedRangeVector bool
```

Comment:
Slight wording inconsistency between `isInstant` and `allowExpectedRangeVector` - could they both start `is` ?
Or maybe the new one could be `expectRangeVector` ?